Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/nhinc/AuditRepositoryOrchImpl.java

```diff
@@ -287,4 +293,21 @@ private Date convertXMLGregorianCalendarToDate(XMLGregorianCalendar xmlCalDate)
         LOG.info("eventDate -> " + eventDate);
         return eventDate;
     }
+
+    private String getRelatesTo(AssertionType assertion) {
+        return NullChecker.isNotNullish(assertion.getRelatesToList()) ? assertion.getRelatesToList().get(0) : null;
+    }
+
+    private String getUserId(List<ActiveParticipant> participants) {
```

Comment:
This method is unnecessary. If we these values are set in LogEventRequestType they can be accessed.